.octopus/deployment_settings.ocl

```diff
@@ -0,0 +1,16 @@
+deployment_changes_template = <<-EOT
+        #{each release in Octopus.Deployment.Changes}
+        #{release.ReleaseNotes}
+        #{/each}
+    EOT

```

Comment:
misaligned whitespace?

Reply:
That's straight out of Octopus after CaC'ifying a project

Reply:
LOL :-)